spec/support/govuk_i18n_cov.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class GOVUKI18nCov
+  module I18n
+    module Registry
+      def lookup(locale, key, scope = [], options = {})
+        GOVUKI18nCov.log(locale, key, scope)
+        super
+      end
+    end
+  end
+
+  class << self
```

Comment:
Note, operating on the eigenclass was discouraged in the GOV.UK style guide before it was replaced. Hence it's seen very little in GOV.UK code. I don't mind it myself, though I wonder if using a Singleton pattern might make it a bit easier to follow as a class.

Reply:
Fair enough - SimpleCov seems to be doing this, but I like it better after moving to the Singleton pattern.